src/controllers/TerminalController.java

```diff
@@ -75,6 +78,53 @@ private void exit() {
         context.exit();
     }
 
+    private String processSpelling(String inputtedCommand) {
+        int maxDistance = (int) Math.ceil(inputtedCommand.length() / 8f);
+        LevenshteinDistance levenshteinDistance = new LevenshteinDistance();
+        int currentMin = Integer.MAX_VALUE;
+        String newCommand = inputtedCommand;
+        for (String command : AllCommands().keySet()) {
+            int dist = levenshteinDistance.apply(command, inputtedCommand);
+            if (dist <= maxDistance) {
```

Comment:
This can be reduced to a single if statement

Reply:
I am using the dist variable in the if statement body, therefore this cannot be reduced.

Reply:
Oh never mind, I understand what you mean, highlighting line 90 would have been great.